pkg/bgpv1/gobgp/conversions.go

```diff
@@ -69,6 +70,7 @@ func ToAgentPath(p *gobgp.Path) (*types.Path, error) {
 		PathAttributes: pattrs,
 		AgeNanoseconds: ageNano,
 		Best:           p.Best,
+		GoBGPPathUUID:  p.Uuid,
```

Comment:
IIRC, we decided to delete this GoBGPPath prefix.

Reply:
You are right, just pushed the change with the rename.